### ISSUE TITLE
FuncRef.call_func not found in 2.1RC2

### ISSUE THREAD

**Issue by mcannamela** [godotengine/godot#6035]
**Operating system or device - Godot version:**osx 10.11.5, ubuntu 14.04

**Issue description** (what happened, and what was expected): Making a FuncRef using funcref() and then trying to call_func() breaks with the error: 
`Invalid call. Nonexistent function 'call_func' in base 'FuncRef'.`

**Steps to reproduce:**
Following runs for me under 2.1RC1 but fails under 2.1RC2, in both osx and ubuntu.
```
extends Node2D

class FuncClosure:
	var _y 
	func _init(y):
		_y = y 
	func call_it(x):
		return x + _y

func _ready():
	var f = FuncClosure.new(1)
	var fref = funcref(f, 'call_it')
		
	var z = fref.call_func(2)
```

**Link to minimal example project** (optional but very welcome):

**Comment by TheoXD** [comment]
broken by https://github.com/godotengine/godot/commit/b80c42ef4e99ee155a98c7a2f17201280612257f#diff-1bd5d382d9201d75ddcd89131b5333c4R68

**Comment by akien-mga** [comment]
CC @eska014

**Comment by Keyaku** [comment]
This breaks *a lot* of projects without warning and should be fixed ASAP :X

**Comment by eska014** [comment]
Fix PR is already available #6055, but should just revert the faulty line for 2.1

**Comment by Keyaku** [comment]
Either that or write `call_func:Variant` from now on :]

**Comment by eska014** [comment]
fixed by #6078 for 2.1 and #6055 for further development
